test/property.test.js

```diff
@@ -125,6 +207,20 @@ describe('loopback:property generator', function() {
     });
   });
 
+  it('should fail to create with unsupported type', function (done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'something',
+      type: 'unknown',
```

Comment:
yes, "the type is picked from a list", but that doesn't mean this code path could not eventually be exercised in the future through other means beside CLI, so I do believe having the error handling in place is safe and not have any adverse impact.